src/TextareaPrompt.php

```diff
@@ -113,7 +113,7 @@ protected function handleUpKey(): void
         $lines = collect($this->lines());
 
         // Line length + 1 for the newline character
-        $lineLengths = $lines->map(fn ($line, $index) => mb_strwidth($line) + ($index === $lines->count() - 1 ? 0 : 1));
+        $lineLengths = $lines->map(fn ($line, $index) => count(mb_str_split($line)) + ($index === $lines->count() - 1 ? 0 : 1));
```

Comment:
Any reason not to use `mb_strlen` instead?

Reply:
Nope 🤦‍♂️

I thought about it but for some reason thought it would return the wrong value. Not... not sure why.

Updated, thank you!